shotgun_api3/lib/mockgun.py

```diff
@@ -328,24 +330,7 @@ def find(self, entity_type, filters, fields=None, order=None, filter_operator=No
             # traditiona style sg filters
             resolved_filters = filters
 
-        # now translate ["field", "in", 2,3,4] --> ["field", "in", [2, 3, 4]]
-        resolved_filters_2 = []
-        for f in resolved_filters:
-
-            if len(f) > 3:
-                # ["field", "in", 2,3,4] --> ["field", "in", [2, 3, 4]]
-                new_filter = [ f[0], f[1], f[2:] ]
-
-            elif f[1] == "in" and not isinstance(f[2], list):
-                # ["field", "in", 2] --> ["field", "in", [2]]
-                new_filter = [ f[0], f[1], [ f[2] ] ]
-
-            else:
-                new_filter = f
-
-            resolved_filters_2.append(new_filter)
-
-        results = [row for row in self._db[entity_type].values() if self._row_matches_filters(entity_type, row, resolved_filters_2, filter_operator, retired_only)]
+        results = [row for row in self._db[entity_type].values() if self._row_matches_filters(entity_type, row, resolved_filters, filter_operator, retired_only)]
```

Comment:
I see this isn't something you actually changed, but wondering if we can clean it up a bit?
```
results = [
    row for row in self._db[entity_type].values() 
        if self._row_matches_filters(
            entity_type, 
            row, 
            resolved_filters, 
            filter_operator, 
            retired_only
        )
]
```
Maybe throw some comments in there too?